
package assignment12;

import java.lang.reflect.Array;
import java.util.Scanner;

public class ExceptionDemo { //i created class EXceptionDemo
	
	public static void main(String [] args){ //created main method
	    
	    System.out.println("Enter a String");
	    String[] Array = {"uma","shanti","navin","prem","satya"};//taken 5 string elements
	    
	    }
private char[][] Size;
	    public void search() throws Exception{//taking a method search element

	    	Scanner sc = new Scanner(System.in);
		    String name=sc.next(); //taken scanner to give input
		    
		  
		    
			System.out.println(Size[1]);//showing how the exception handles
		    System.out.println("value found");
		     
		   try{   
	    }
	    catch (Exception e){
	    	
	    return;
	    
	    }   
	    }
}


